lib/index.js

```diff
@@ -61,53 +62,7 @@ var search = function (opts) {
 
 /* istanbul ignore if  */
 if (require.main === module) {
```

Comment:
Can you move this whole stanza to the new file? This doesn't need to be in this one. This if statement just says "If this is imported, don't run this. If it's being run via the `node` executable, do these things.